src/localdb/parsetools/value.rs

```diff
@@ -0,0 +1,222 @@
+use uuid::Uuid;
+
+use super::Value;
+
+pub trait ValueParser {
+    fn try_i8(&self) -> Result<i8, ParseValueError>;
+    fn possible_i8(&self) -> Result<Option<i8>, ParseValueError>;
+    fn try_i16(&self) -> Result<i16, ParseValueError>;
+    fn possible_i16(&self) -> Result<Option<i16>, ParseValueError>;
+    fn try_i32(&self) -> Result<i32, ParseValueError>;
+    fn possible_i32(&self) -> Result<Option<i32>, ParseValueError>;
+    fn try_i64(&self) -> Result<i64, ParseValueError>;
+    fn possible_i64(&self) -> Result<Option<i64>, ParseValueError>;
+    fn try_i128(&self) -> Result<i128, ParseValueError>;
+    fn possible_i128(&self) -> Result<Option<i128>, ParseValueError>;
+    fn try_f64(&self) -> Result<f64, ParseValueError>;
+    fn possible_f64(&self) -> Result<Option<f64>, ParseValueError>;
+    fn try_bool(&self) -> Result<bool, ParseValueError>;
+    fn possible_bool(&self) -> Result<Option<bool>, ParseValueError>;
+    fn try_string(&self) -> Result<String, ParseValueError>;
+    fn possible_string(&self) -> Result<Option<String>, ParseValueError>;
+    fn try_bytea(&self) -> Result<Vec<u8>, ParseValueError>;
+    fn possible_bytea(&self) -> Result<Option<Vec<u8>>, ParseValueError>;
+    fn try_uuid(&self) -> Result<Uuid, ParseValueError>;
+    fn possible_uuid(&self) -> Result<Option<Uuid>, ParseValueError>;
+}
+
```

Comment:
Possible to simplify `ValueParser` with `Parsable`?

Reply:
I'm sorry, I'm a bit confused, do you mean I should use the [Parsable crate](https://docs.rs/parsable/latest/parsable/#)? Or maybe that I should rename the Trait? Or combine the traits used for parsing the GlueSQL payload?

Reply:
First rename the trait,  then study postgres' FromSQL and ToSQL traits

Reply:
Renames pushed to local-storage branch, will study after I finish the other changes

Thanks for the tips!